### ISSUE TITLE
Save jsfiddle source as separate files

### ISSUE THREAD

**Issue by cronfy** [facundovictor/jsfiddle-downloader#7]
When jsfiddle is downloaded, only the Result is stored in output html file.

But this is not enough to make a backup of a fiddle. Is it possible to save HTML, JS and CSS tabs in separate files for a fiddle?

**Comment by facundovictor** [comment]
Right now, the output html file contains:
- All the html included on the HTML tab.
- All external imports.
- All the JS included on the JS tab:
```
<script type="text/javascript">
    /* Here is included all the js code from the JS tab. */
</script>
```
- All the JS included on the CSS tab:
```
  <style type="text/css">
    /* Here is included all the css code from the CSS tab. */
  </style>
```

It contains all the code and is enough for doing a backup (at least it was for me when I created this tool). Nevertheless, saving it in different files would be a nice feature to add.

**Comment by cronfy** [comment]
> All the JS included on the CSS tab:

When there is SCSS in CSS tab, it is not saved - compiled CSS is saved instead. This is the issue.

**Comment by facundovictor** [comment]
@cronfy, you're right, this doesn't support SCSS! But, it should be tracked in a separate Issue --> [Issue Nº8](https://github.com/facundovictor/jsfiddle-downloader/issues/8). Thanks for reporting this! I'll be working on this features. If you also want to collaborate, PRs are welcome.
